src/main/java/org/sonarlint/intellij/ui/nodes/SummaryNode.java

```diff
@@ -61,10 +66,13 @@ public String getText() {
     }
 
     if (forSecurityHotspot) {
-      return String.format("Found %d %s in %d %s", findings, findings == 1 ? "Security Hotspot" : "Security Hotspots", files, files == 1 ? "file" : "files");
+      return String.format(FORMAT, findings, findings == 1 ? "Security Hotspot" : "Security Hotspots", files, files == 1 ? "file" : "files");
     }
 
-    return String.format("Found %d %s in %d %s", findings, findings == 1 ? "issue" : "issues", files, files == 1 ? "file" : "files");
+    if(forOldIssues){
+      return String.format(FORMAT, findings, findings == 1 ? "issue" : "issues", files, files == 1 ? "file since old analysis" : "files since old analysis");
+    }
+    return String.format(FORMAT, findings, findings == 1 ? "issue" : "issues", files, files == 1 ? "file since new analysis" : "files since new analysis");
```

Comment:
I think you can set a `String` for the text `since old/new analysis`, and just return this `String` inside this line, so it avoids having duplication and the `if (forOldIssues)`

Reply:
Since those text will change after backend integration I think it is not needed for now

Reply:
I think it will be the same after, something like this:
`var sinceText = forOldIssues ? "since old analysis" : "since new analysis";`
`return String.format(FORMAT, findings, findings == 1 ? "issue" : "issues", files, files == 1 ? "file " + sinceText : "files " + sinceText);`

Later, we'll just replace "old analysis" by whatever we retrieve from core